Rlabkey/DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: Rlabkey
-Version: 2.8.1
-Date: 2021-09-10
+Version: 2.8.2
+Date: 2021-11-30
```

Comment:
The updated release date and version should also get updated in the `Rlabkey-package.Rd` file.